code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -155,6 +155,72 @@
 	else
 		..()
 
+/mob/living/simple_animal/friendly/sheep
+	name = "sheep"
+	desc = "It's so fluffy! can't shear it though. milking is fine."
+	icon_state = "sheep"
+	icon_living = "sheep"
+	icon_dead = "sheep_dead"
+	icon_gib = "cow_gib"
+	speak_emote = list("baas","baas hauntingly")
+	turns_per_move = 5
+	see_in_dark = 6
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	health = 50
+
+	meat_type = /obj/item/reagent_containers/food/snacks/meat
+	meat_amount = 4
+	bone_amount = 6
+	skin_material = MATERIAL_SKIN_COWHIDE
+	skin_amount = 4
+
+	var/datum/reagents/udder = null
+
+	ai_holder_type = /datum/ai_holder/simple_animal/passive/cow
+	say_list_type = /datum/say_list/cow
+
+/mob/living/simple_animal/friendly/sheep/New()
+	udder = new(50, src)
+	..()
+
+/mob/living/simple_animal/friendly/sheep/attackby(var/obj/item/O as obj, var/mob/user as mob)
```

Comment:
The `var/` and `as obj`/`as mob` in these arguments are unnecessary. You could do `obj/item/O, mob/user` and it'll work just fine. It's cleaner that way, so I'd recommend it.